docs/how-to-build-a-lexical-model.md

```diff
@@ -0,0 +1,365 @@
+  </Options>
+  <Info>
+    <Name URL="">SENĆOŦEN (Saanich Dialect) Lexical Model</Name>
+    <Copyright URL="">© 2019 National Research Council Canada</Copyright>
+    <Author URL="mailto:Eddie.Santos@nrc-cnrc.gc.ca">Eddie Antonio Santos</Author>
+    <Version>1.0.3</Version>
+  </Info>
+  <Files>
+    <File>
+      <Name>..\build\nrc.str.sencoten.model.js</Name>
+      <Description>Lexical model nrc.str.sencoten.model.js</Description>
+      <CopyLocation>0</CopyLocation>
+      <FileType>.model.js</FileType>
+    </File>
+  </Files>
+  <LexicalModels>
+    <LexicalModel>
+      <Name>SENĆOŦEN dictionary</Name>
+      <ID>nrc.str.sencoten</ID>
+      <Version>1.0.3</Version>
+      <Languages>
+        <Language ID="str">North Straits Salish</Language>
+        <Language ID="str-Latn">SENĆOŦEN</Language>
+      </Languages>
+    </LexicalModel>
+  </LexicalModels>
+</Package>
+```
+
+Step 4.3: Creating the model file
+---------------------------------
+
+Finally, the `model.ts` is a TypeScript source code file that is
+executed to build the model. For our purposes, we can simply copy-paste
+the following example:
+
+```typescript
+import LexicalModelCompiler from "@keymanapp/developer-lexical-model-compiler";
+
+(new LexicalModelCompiler).compile({
+  format: 'trie-1.0',
+  wordBreaking: 'default',
+  sources: ['wordlist.tsv'],
+});
+```
+
+Most importantly, this file specifies that model will use the word list
+in `wordlist.tsv`.
+
+For simple lexical models with exactly one wordlist called
+`wordlist.tsv`, **this file does not need to be changed any further**.
+
+However, it specifies a few other things:
+
+  - `format: 'trie-1.0'`: Use the trie back-end, that allows for
+    prediction based on prefixes of words.
+  - `wordBreaking: 'default'`: use Unicode's default word boundary rules
+    to find where "words" are when predicting text. This works for most
+    languages and writing systems, however, it does not work for
+    languages/writing systems that do not mark breaks between words,
+    e.g., Chinese, Thai, Laos, Khmer, and Japanese, to name a few. For
+    these writing systems, you must provide a language-specific word
+    breaking algorithm to find the words in running text.
+
+Step 5: Build the model
+-----------------------
+
+
+Appendix: Tab-separated wordlist file format specification
+----------------------------------------------------------
+
+<a id="appendix:tsv"></a>
+
+[appendix]: ./#appendix:tsv
+
+The **lexical model compiler** expects wordlists to abide by the
+following **tab-separated values** (TSV) format:
+
+ - the file is must be **UTF-8** encoded text file
+ - newlines are either **LF** or **CRLF**
+ - the file either consists of a **comment** or an **entry**
+ - **comment** lines MUST start with the '#' character on the very first column
+ - **entries** are one or two columns, separated by the (horizontal)
+   **tab character** (U+0009)
+ - column 1 (REQUIRED): the **word form**: can have any character
```

Comment:
Suggest noting that quote characters are not used to surround the text?

Reply:
Correct!